code/modules/power/apc.dm

```diff
@@ -955,11 +965,15 @@
 	. = ..()
 	if (!. && !QDELETED(remote_control))
 		. = remote_control.can_interact(user)
+	if (hasSiliconAccessInArea(user,area))
+		return TRUE
```

Comment:
nope, it will only do it if the APC is in the same area it controls, which it usually isn't, so you need to be able to control the APC even if you don't have conrol of the area it's in

Reply:
Thing is this also makes silicons capable of interacting with APCs from any distance.